### ISSUE TITLE
Add lang attribute to HTML tag

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#360]
In order for users and machines to know what human language the site is in, there should be a lang attribute on the HTML element.

- [x] The html element should have a lang of en

>
The html element should have a lang or xml:lang attribute which describes the language of the document.

**Comment by msecret** [comment]
Solution: `<html lang="en">`
